src/TestRoute.js

```diff
@@ -31,6 +31,7 @@ export default class TestRoute extends React.Component {
                 <input
                   type='text'
                   name='path'
+                  size='82'
```

Comment:
Instead of specifying an exact size, I think that it would be better to use a percentage width. It doesn't have to be styled perfectly, but something that is semi-responsive to the screen size.
```js
style={{ width: '85%' }}
```